ui/perfherder/alerts/Assignee.jsx

```diff
@@ -33,7 +33,9 @@ export default class Assignee extends React.Component {
         inEditMode: true,
         // input prefills with this field, so
         // we must have it prepared
-        newAssigneeUsername: assigneeUsername,
+        newAssigneeUsername: assigneeUsername
+          ? assigneeUsername.split('/')[1]
```

Comment:
I don't think you need to do that because on line 70 you already set the username correctly when using the "Take" functionality. For the case where the user fills in the information themselves this is not needed because they are required to only fill in the email. Let me know what you think.

Reply:
This line is necessary when we want to reassign an alert. If that line doesn't exist, when enter in edit mode on an already assigned alert, we will see 'mozilla-ldap/fbilt@mozilla.com' instead of 'fbilt@mozilla.com'